### PR TITLE
#28675 -Postman tests refactor in ContentResourceV1 Postman Tests

### PR DESCRIPTION
**Pull request by bryanboza**
### Proposed Changes
- Migration to JWT auth 
- Group request into folders
- New scripts for tests 

### Checklist
- [ ] Tests
- [ ] Security Implications Contemplated (JWT variables as secrets)

### CODE REVIEW

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json [hunk 1]
```diff
@@ -207,1437 +161,1330 @@
...
+									],
+									"path": [
+										"api",
+										"v1",
+										"contenttype",
+										"id",
+										"ParentContentType"
+									]
+								},
+								"description": "We have to update the parent content type to include the child relationship when the child ct already exits"
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json:605]: exits --> exists

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json [hunk 2]
```diff
@@ -207,1437 +161,1330 @@
...
+										"api",
+										"v1",
+										"workflow",
+										"actions",
+										"default",
+										"fire",
+										"PUBLISH"
+									]
+								},
+								"description": "Create the a parent content instance"
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json:659]: Create the **a** parent content instance

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json [hunk 3]
```diff
@@ -207,1437 +161,1330 @@
...
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json:713]: Create the **a** parent content instance

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json [hunk 4]
```diff
@@ -207,1437 +161,1330 @@
...
-			],
-			"request": {
-				"auth": {
-					"type": "basic",
-					"basic": [
-						{
-							"key": "password",
-							"value": "admin",
-							"type": "string"
+							"name": "Save Multiple Childs for parent 1",
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json:718]: multiple _children_

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json [hunk 5]
```diff
@@ -207,1437 +161,1330 @@
...
+										"api",
+										"v1",
+										"workflow",
+										"actions",
+										"default",
+										"fire",
+										"PUBLISH"
+									]
+								},
+								"description": "Save all child for parent1"
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json:781]: _children_

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json [hunk 6]
```diff
@@ -207,1437 +161,1330 @@
...
+							},
+							"response": []
 						},
 						{
-							"key": "username",
-							"value": "admin@dotcms.com",
-							"type": "string"
+							"name": "Save Multiple Childs for parent 2",
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json:786]: _children_

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json [hunk 7]
```diff
@@ -207,1437 +161,1330 @@
...
+										"api",
+										"v1",
+										"workflow",
+										"actions",
+										"default",
+										"fire",
+										"PUBLISH"
+									]
+								},
+								"description": "Save all childs for parent2"
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json:843]: _children_

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json [hunk 8]
```diff
@@ -207,1437 +161,1330 @@
...
-							"value": "admin",
-							"type": "string"
+							"name": "Related - Non Existing id",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"Correct message\", function () {",
+											"    pm.expect(pm.response.text()).to.include(\"The identifier:non-existing does not exists\");",
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json:1318]: _exist_

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json [hunk 9]
```diff
@@ -207,1437 +161,1330 @@
...
+										"{{serverURL}}"
+									],
+									"path": [
+										"api",
+										"v1",
+										"content",
+										"related"
+									]
+								},
+								"description": "Test the related contentlets with relationship field var that do not exists, should return 400"
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json:1397]: _exist_

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json [hunk 10]
```diff
@@ -207,1437 +161,1330 @@
...
-						],
-						"type": "text/javascript"
-					}
-				}
-			],
-			"request": {
-				"auth": {
-					"type": "basic",
-					"basic": [
+					"description": "This folder contains a set of requests designed to ensure that our API correctly handles attempts to relate content using invalid or non-existent IDs. The validations within these requests are essential to maintaining the integrity and consistency of our data relationships.\n\n#### Objectives:\n\n1. **Invalid ID Detection**: Verify that the API appropriately rejects attempts to relate content using invalid IDs.\n    \n2. **Error Messaging**: Ensure that the API provides clear and informative error messages when invalid IDs are used.\n    \n3. **Data Integrity**: Confirm that no data relationships are established or altered when invalid IDs are submitted.\n    \n4. **Edge Cases**: Test various edge cases, including empty IDs, improperly formatted IDs, and IDs that do not exist in the database.",
```
Comment (nollymar) [inline dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json:1402]: _to maintain_

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json (hunk 2 above)
Comment (bryanboza) [reply dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json]: done!

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json (hunk 3 above)
Comment (bryanboza) [reply dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json]: done!

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json (hunk 4 above)
Comment (bryanboza) [reply dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json]: done!

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json (hunk 5 above)
Comment (bryanboza) [reply dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json]: done!

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json (hunk 6 above)
Comment (bryanboza) [reply dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json]: done!

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json (hunk 7 above)
Comment (bryanboza) [reply dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json]: done!

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json (hunk 8 above)
Comment (bryanboza) [reply dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json]: done!

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json (hunk 9 above)
Comment (bryanboza) [reply dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json]: done!

File: dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json (hunk 10 above)
Comment (bryanboza) [reply dotcms-postman/src/main/resources/postman/ContentResourceV1.postman_collection.json]: AI change this to: "for maintaining"

**Review by jdotcms — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Review by nollymar — APPROVED** [review]

**Pull request merged**